.github/workflows/update-gemfiles.yml

```diff
@@ -3,78 +3,57 @@ name: Update Gemfiles
 # This action cannot be skipped altogether because it is a mandatory status check.
 # Instead we conditionally skip it at job level, instead of workflow level.
 on:
-  # Execute on `push` and not `pull_request` because `pull_request`
```

Comment:
Ahh interesting, I wasn't aware of this workflow - would this have been getting called and regenerating all of the `gemfile/` directory?